xml/System/Memory`1.xml

```diff
@@ -384,11 +467,17 @@
         <Parameter Name="length" Type="System.Int32" />
       </Parameters>
       <Docs>
-        <param name="start">To be added.</param>
-        <param name="length">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="start">The index at which to begin the slice.</param>
+        <param name="length">The number of elements to include in the slice.</param>
+        <summary>Forms a slice out of the current memory starting at a specified index for a specified length.</summary>
+        <returns>An object that <paramref name="length" /> elements from the current instance starting at <paramref name="start" />. </returns>
```

Comment:
Sentence is awkward to read

Reply:
Summary really explains it best. Return should just be shortened to something like "Memory object contain X elements from the current instance.